atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockIntegrationTest.java

```diff
@@ -101,7 +101,8 @@ private void setUpWithCasSupportSetTo(boolean supportsCas) throws Exception {
                 CassandraKeyValueServiceConfigManager.createSimpleManager(quickTimeoutConfig);
         TracingQueryRunner queryRunner = new TracingQueryRunner(log, TracingPrefsConfig.create());
         writeConsistency = ConsistencyLevel.EACH_QUORUM;
-        clientPool = CassandraClientPoolImpl.create(simpleManager.getConfig());
+        clientPool = CassandraClientPoolImpl.create(simpleManager.getConfig(),
+                TokenRangeWritesLogger.createUninitialized());
```

Comment:
Why does every call to `create` need to pass this in?  I think we minimise the impact of the change by simply creating the `TokenRangeWritesLogger`  in the factory method and then only passing it in for tests.

Reply:
I refactored it now. Had to add to the bloat of the CassandraClientPool interface, but other than that much cleaner now.